docs/overview_rs.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+---
+
+# Overview_RS
+
+The To-Do service is a cloud-based platform that offers subscribers a task list
+where they can post tasks and receive timely reminders. Our service provides the REST
```

Comment:
Just a tip: If you start each sentence in a new line, they're easier to review and see changes as you edit them in the future.